src/Stats/index.jsx

```diff
@@ -51,12 +53,16 @@ export default function Stats() {
       <ScrollView style={styles.deviceBody}>
         <View style={{ marginBottom: 20 }}>
           <View style={{ flexDirection: 'row', justifyContent: 'space-between' }}>
-            <FilterBtn />
+            <FilterBtn
+              transFilter={transFilter}
+              setTransFilter={setTransFilter}
```

Comment:
this property should be `onFilterChnage` and we shouldn't pass setTrandFilter to the component, we should call it in this component instead
```js
onFilterChange={(newFilter) => setTransFilter(newFilter)}
```

Reply:
fixed